pages/common/sops.md

```diff
@@ -0,0 +1,29 @@
+# sops
+
+> SOPS: Secrets OPerationS.
+> Simple and flexible tool for managing secrets.
+> More information: <https://github.com/mozilla/sops>.
+
+- Encrypt a file:
+
+`sops -e {{path/to/myfile.json}} > {{path/to/myfile.enc.json}}`
+
+- Decrypt a file in `cat` fashion using the `-d` flag:
+
+`sops -d {{path/to/myfile.enc.json}}`
+
+- Rotate data keys for a sops file:
+
+`sops -r {{path/to/myfile.enc.yaml}}`
+
+- Change the extension of the file once encrypted:
+
+`sops -d --input-type json {{path/to/myfile.enc.json}}`
+
+- Extract keys by naming them, and array elements by numbering them:
+
+`sops -d --extract '["an_array"][1]' {{path/to/myfile.enc.json}}`
+
+- Diff two sops files:
```

Comment:
```suggestion
Show the difference between two sops files
```

Reply:
You missed the `-` and the`:`, so I added then for you :-)